ARK-44 List Header and ARK-43 Author List (#41)
JesusFilm/Ark: src/elements/list-header/list-header.docs.mdx

---
description: 'A ListHeader component.'
labels: ['Lists', 'React', 'Typescript']
---

import { ListHeader } from './list-header';

## React Component our the list header

> A component that renders the header of a list it's under in

Reviewer: ```suggestion
A component that renders a header for a list of items
```

> ### Using props to customize the text

Reviewer: ```suggestion
### Using props to customize the title
```
